tools/im2rec.py

```diff
@@ -11,7 +11,6 @@
 import cv2
 import time
 import traceback
-from builtins import range

```

Comment:
In Python 2.x, need to install the "future" package to use the "builtins" function. so this line was removed.